### ISSUE TITLE
New Relic - Real User Monitoring (RUM) support (documentation)

### ISSUE THREAD

**Issue by dlehammer** [agorapulse/grails-newrelic#11]
I've been reading the plugin documentation, `README.md`, in-order to understand the requirements for achieving the same default analysis-data as for Grails 2, ie. transactions and service analysis.
And seemingly it's simple, but I would like to be sure - and as a side-effect improve the documentation.

> Once installed, New Relic Browser will need to be configured for [manual instrumentation](https://docs.newrelic.com/docs/agents/java-agent/instrumentation/page-load-timing-java#manual_instrumentation)

My impression upon reading the link, is that this step is only required for adding RUM-instrumentation.

> [Usage](https://github.com/agorapulse/grails-newrelic#usage)

My impression upon reading the section, is that this is only required for RUM-instrumentation.

I suggest the RUM-specific documentation is encapsulated in a dedicated section.

**Comment by musketyr** [comment]
hi, thanks for the feedback! do you think you can summarize your findings into a PR?

**Comment by dlehammer** [comment]
Hi @musketyr,

I was/am looking for confirmation/review of my findings - as I have no experience with RUM-instrumentation.

Ie. originally I came here looking for the simplest way of restoring New Relic analytics after migrating from Grails 2 to 3 ([pending official support](https://discuss.newrelic.com/t/feature-idea-grails-3-support/37362)).
And  I got confused by the documentation, and I wanted to help sort it out :)
